ui-particles-angular/projects/ui-policy-studio-angular/src/lib/components/flow-execution-form-dialog/gio-ps-flow-execution-form-dialog.component.html

```diff
@@ -0,0 +1,56 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<h2 mat-dialog-title class="title">
+  Flow execution
+
+  <button class="title__closeBtn" mat-icon-button aria-label="Close dialog" [mat-dialog-close]="false">
+    <mat-icon svgIcon="gio:cancel"></mat-icon>
+  </button>
+</h2>
+
+<mat-dialog-content *ngIf="flowExecutionFormGroup" class="content" [formGroup]="flowExecutionFormGroup">
+  <div class="content__row">
+    <gio-banner-info>
+      By default, flow selection is based on the operators defined in your flows, either when it “matches with” or “starts with” a path. You
+      can choose the "Best match" option to select a flow from the closest path.
+    </gio-banner-info>
+  </div>
+  <div class="content__row">
+    <mat-form-field>
+      <mat-label>Default flow mode</mat-label>
+      <mat-select formControlName="mode">
+        <mat-option value="BEST_MATCH">Best Match</mat-option>
+        <mat-option value="DEFAULT">Default</mat-option>
```

Comment:
question: shouldn't the displayed value be "Starts with" here ?

Reply:
No because Default and Best Match use the path-operator to filter flows
![image](https://github.com/gravitee-io/gravitee-ui-particles/assets/13161768/8ed7f94b-7bb4-4038-b3b4-d520abcca5a4)

The difference is after. `Default` uses all flows, `Best Match` uses only the one that is closer to the requested URL